packages/core/bootstrap/src/lib/server.ts

```diff
@@ -124,3 +130,29 @@ function setupMetricsServer() {
 
   metricsApp.listen(metricsPort, () => logger.info(`Monitoring listening on port ${metricsPort}!`))
 }
+
+function initExpressMiddleware(
+  app: express.Express,
+  // context: AdapterContext
+) {
+  app.set('trust proxy', 1)
+
+  // TODO: utilize context for Redis
+
+  const limiter = rateLimit({
+    windowMs: 5 * 100, // 5 seconds
+    max: 250, // limit each IP to 250 requests per windowMs
+    keyGenerator: () => '*', // use one key for all incoming requests
+  })
+  app.use(limiter)
+
+  const speedLimiter = slowDown({
```

Comment:
Also, these values should always be smaller than the rate limiter, no? Otherwise, the rate limiter will always take precedence and `429` requests before the slow down limiter can do anything. Maybe we should configure the `delayAfter` value to be proportional (80%? 70%?) to the `max` value of the rate limiter. This would give us the following variable set:
```ts
const windowMs = 1000 * 60 // define window as per minute, since this is a common interval
const max = parseInt(process.env.EA_RATE_LIMIT_MAX) || 1500 // default to 1500 req/min max
const delayAfter = max * ( Number(process.env.EA_SLOW_DOWN_AFTER_FACTOR) || 0.8) // we start slowing down requests when we reach 80% of our max limit for the current interval
const delayMs = parseInt(process.env.EA_SLOW_DOWN_DELAY_MS) || 250 // default to slowing down each request by 250ms
```